rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -3845,6 +3850,154 @@ extern void ppu_precompile(std::vector<std::string>& dir_queue, std::vector<ppu_
 				}
 			}
 		}
+
+		if (file_queue.size() > old_size && file_queue.size() - 1 > old_size)
+		{
+			// Metal Gear Solid 4 has multiple executables for different languages, and they are massive each!
+			// The difference between them is two letters, abbrevation of the language
```

Comment:
Why isn't there any example in the comment?
Are we supposed to read the entire code to understand the simple syntax?

Reply:
Added examples